package.json

```diff
@@ -50,33 +47,36 @@
     "querystring": "qs"
   },
   "dependencies": {
-    "@types/chai": "4",
-    "@types/superagent": "^8.1.6",
     "charset": "^1.0.1",
     "cookiejar": "^2.1.4",
-    "is-ip": "^3.1.0",
+    "is-ip": "^5.0.1",
     "methods": "^1.1.2",
-    "qs": "^6.11.2",
+    "qs": "^6.12.1",
     "superagent": "^9"
   },
   "devDependencies": {
-    "@jsdevtools/simplifyify": "^8.0.4",
     "@semantic-release/changelog": "^6",
     "@semantic-release/commit-analyzer": "^12.0.0",
     "@semantic-release/git": "^10",
     "@semantic-release/npm": "^12.0.0",
     "@semantic-release/release-notes-generator": "^13.0.0",
-    "chai": "4",
+    "@types/chai": "^4.3.15",
+    "@types/superagent": "^8.1.7",
```

Comment:
Upgrading from `4.3.0` to `5.1.1` broke our typechecking. Code like this

```ts
chai.request
  .agent(this._server)
  .post('/api/preview')
```

now fails with this error:

```
error TS2339: Property 'post' does not exist on type 'Agent'.
```

We think it's because `@types/superagent` is now in `devDependencies`, instead of `dependencies`. I think it should be in `dependencies`, because your package exports `superagent` types publicly. Since the superagent types are part of your public types, any package which uses your types should also include the `superagent` types that you export.

Reply:
Can you open a standalone issue for this? 

It sounds like you're right but I'll need to look when I'm not mobile and can test it out a bit

Reply:
Yep! Thank you for building and maintaining this package!

* https://github.com/chaijs/chai-http/issues/346